client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -318,4 +325,87 @@ public void onFailure(Exception e) {
             // end::delete-conflict
         }
     }
+
+    public void testBulk() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        {
+            BulkRequest bulkRequest = new BulkRequest()
+                    .add(new IndexRequest("index", "type", "1").source(Requests.INDEX_CONTENT_TYPE, "field", "value"))
+                    .add(new IndexRequest("index", "type", "2").source(Requests.INDEX_CONTENT_TYPE,"field", "value"));
+            BulkResponse bulkResponse = client.bulk(bulkRequest);
+            assertFalse(bulkResponse.hasFailures());
```

Comment:
maybe this could also be made a first docs snippet with only indexing operation, which is the most common case? just an idea